src/integration-tests/introspector/wl-create-domain-pod.pyt

```diff
@@ -243,6 +243,25 @@ else:
   cmo.setCluster(cl)
   print('Done setting attributes for Server Template: %s' % templateName);
 
+  templateName = '${CLUSTER_NAME}' + "-template-dummy1"
+  print('Creating Server Template: %s' % templateName)
+  st1=create(templateName, 'ServerTemplate')
+  print('Done creating Server Template: %s' % templateName)
+  cd('/ServerTemplates/%s' % templateName)
+  cmo.setListenPort(${MANAGED_SERVER_PORT})
+  #cmo.setListenAddress('${DOMAIN_UID}-${MANAGED_SERVER_NAME_BASE}${id}') # subst-ignore-missing
+  #cmo.setCluster(cl)
+  print('Done setting attributes for Server Template: %s' % templateName);
+ 
+  templateName = '${CLUSTER_NAME}' + "-template-dummy2"
+  print('Creating Server Template: %s' % templateName)
+  st1=create(templateName, 'ServerTemplate')
+  print('Done creating Server Template: %s' % templateName)
+  cd('/ServerTemplates/%s' % templateName)
+  cmo.setListenPort(${MANAGED_SERVER_PORT})
+  #cmo.setListenAddress('${DOMAIN_UID}-${MANAGED_SERVER_NAME_BASE}${id}') # subst-ignore-missing
+  #cmo.setCluster(cl)
+  print('Done setting attributes for Server Template: %s' % templateName);
```

Comment:
does introspectTest.sh need to be updated to check for these additions?

Reply:
No.  These 'unused template' additions are simply a way to test the failure that this pull is fixing, and the change to introspectDomain.py teaches the introspector to handle the unused templates.